packages/grid/src/vaadin-grid.js

```diff
@@ -473,23 +468,6 @@ class Grid extends ElementMixin(
     }
   }
 
-  /**
-   * Override an observer from `DisabledMixin` to not
-   * set `tabindex` on the grid when it is re-enabled.
-   *
-   * @param {boolean} disabled
-   * @param {boolean} oldDisabled
-   * @protected
-   * @override
-   */
-  _disabledChanged(disabled, oldDisabled) {
-    super._disabledChanged(disabled, oldDisabled);
-
-    if (oldDisabled) {
-      this.removeAttribute('tabindex');
```

Comment:
**note:** The `tabindex` attribute is now removed by `TabIndexMixin` that has become possible since `_lastTabIndex` became `undefined` by default.